api/src/services/email/index.ts

```diff
@@ -24,8 +24,36 @@ export type EmailService = {
 
   resetPassword:
     (c: Config, ap: AppEnum, a: User, t: string) => Promise<void>
+
+  webhooks: {
+    onHerokuStagingRelease: (c: Config, appName: string) => Promise<void>
+  }
 };
 
+const webhooks: EmailService['webhooks'] = {
+  async onHerokuStagingRelease (cfg, appName) {
+    await EmailDispatcher.sendBatch(cfg, [
+      {
+        from: cfg.email.fromAddress,
+        to: cfg.email.developers[0],
+        templateId: WebhookEmailTemplates.ON_HEROKU_RELEASE,
+        templateModel: {
+          app_name: appName,
+          product_name: 'Twine Platform',
+        },
+      },
+      {
+        from: cfg.email.fromAddress,
+        to: cfg.email.developers[1],
```

Comment:
Is there any reason we only ever want to send two emails here? If not, suggest we just `map` over the emails?